### PR TITLE
Whitelisting HPRO app IDs, Vibrent IPs for use in RDR in our non-prod

### PR DESCRIPTION
**Pull request by danqrodney**
environments.

(Once we verify this works, we can configure prod as well.)

### CODE REVIEW

File: rest-api/config/config_test.json [hunk 1]
```diff
@@ -7,10 +7,17 @@
...
     "pmi-hpo-dev@appspot.gserviceaccount.com": {
       "roles": ["healthpro"],
-      "comment": "for HealthPro testing"
+      "comment": "for HealthPro testing",
+      "whitelisted_appids": ["pmi-hpo-dev"],
     },
     "vibrent-drc-qa@vibrent-drc-qa-165017.iam.gserviceaccount.com": {
       "roles": ["ptc"]
+      "whitelisted_ip_ranges": {
+        "ip4": ["34.206.2.13",
```
Comment (calbach) [inline rest-api/config/config_test.json]: For my understanding: what do these IP addresses correspond to? Are they stable? How would one regenerate this list?

**Comment by danqrodney** [comment]
Reverted changes to test, as we want to be able to connect to it from local dev machines.

File: rest-api/config/config_stable.json [hunk 2]
```diff
@@ -6,10 +6,14 @@
   ],
   "user_info": {
     "pmi-hpo-test@appspot.gserviceaccount.com": {
-      "roles": ["ptc", "healthpro"]
+      "roles": ["ptc", "healthpro"],
```
Comment (jt2k) [inline rest-api/config/config_stable.json]: While we're updating this, I don't think there's a reason for this user to have the `ptc` role.

File: rest-api/config/config_test.json (hunk 1 above)
Comment (danqrodney) [reply rest-api/config/config_test.json]: They are IP addresses Vibrent sent us. They are supposed to be stable. If they change, Vibrent would have to tell us.

File: rest-api/config/config_stable.json (hunk 2 above)
Comment (danqrodney) [reply rest-api/config/config_stable.json]: Removed.

(In the distant past, HPRO could create participants via a form and needed to act like PTC, which I'm guessing is why this was here. I presume we don't want to support such shenanigans anymore. :)

**Comment by danqrodney** [comment]
Added healthpro-beta to staging app ID whitelist.

**Pull request merged**